core/src/main/scala/kafka/network/SocketServer.scala

```diff
@@ -1310,6 +1312,7 @@ class ConnectionQuotas(config: KafkaConfig, time: Time, metrics: Metrics) extend
   // Listener counts and configs are synchronized on `counts`
   private val listenerCounts = mutable.Map[ListenerName, Int]()
   private[network] val maxConnectionsPerListener = mutable.Map[ListenerName, ListenerConnectionQuota]()
+  private val TotalConnectionCountMetricName = "totalConnectionCount"
```

Comment:
Can you make the metric name more meaningful? i.e., indicating what connection count, such as TotalBrokerClientConnectionCount. And I think the current convention is using capital letter for the first letter.

Reply:
Hi Huilin, thanks for the comment. I think "TotalConnectionCount" here might be more concise and follow the naming conversion of the relevant configs. E.g. in Kafka, there are such configs:

val MaxConnectionsPerIp: Int = Int.MaxValue
val MaxConnections: Int = Int.MaxValue
val MaxConnectionCreationRate: Int = Int.MaxValue